migration-guide/business-central.adoc

```diff
@@ -0,0 +1,129 @@
+include::../partials/attributes.adoc[]
+= Migrating from Business Central to IBM {DEV_TOOLS}
+
+== Overview
+
+Business Central was a web-based platform for authoring business automation assets such as data objects, rules, processes, cases, and planning entities. It included capabilities for monitoring, administration, and management of business automation assets.  With the release of IBM {DEV_TOOLS} as well as IBM {CANVAS}, which unlike Business Central was designed from the ground up to be cloud-native, most of these asset management capabilities have been replaced or in some cases take a different approach altogether.  This section covers migration of business assets created in Business Central so that they are able to be maintained by developers using either {DEV_TOOLS} and/or {CANVAS}.
```

Comment:
@blulas - 

1. Replace 'such as' with 'like' - >As per IBM standards & guidelines
2. One sentence per line as ASCII doc standards
3. We can keep it as 'With the release of IBM {DEV_TOOLS} and IBM {CANVAS}' instead of 'With the release of IBM {DEV_TOOLS} as well as IBM {CANVAS}'
4. 'is' missing between 'which unlike'
5. Not clear ...'Business Central was designed from the ground up to be cloud-native' - >May be we can keep it simple
6. Most of 'these' asset management capabilities 'have been' replaced or 'in some cases' take a different approach altogether- > Most of 'the' asset management capabilities 'were' replaced or 'sometimes' take a different approach altogether. (This sentence is too long, may be we can keep it short).
7. This section 'covers' migration of business assets created in Business Central so that they are able to be maintained by developers using either {DEV_TOOLS} and/or {CANVAS} -> This section describes the migration of business assets created in Business Central to enhance the developers to use either {DEV_TOOLS} or {CANVAS}.